docs/topics/testing/tools.txt

```diff
@@ -1615,8 +1615,7 @@ your test suite.
             '<input id="id_accept_terms" type="checkbox" checked>'
         )
 
-    ``html1`` and ``html2`` must be valid HTML. An ``AssertionError`` will be
-    raised if one of them cannot be parsed.
+    ``html1`` and ``html2`` are strings that contains HTML.
```

Comment:
I think it's enough to remove `valid`, e.g.:
```suggestion
    ``html1`` and ``html2`` must contain HTML. An ``AssertionError`` will be
    raised if one of them cannot be parsed. 
```
Also, the second sentence should stay, see:
https://github.com/django/django/blob/6993e1bf6dff04b6e3ab284b154397b341463b50/tests/test_utils/tests.py#L877-L889